eclipse-scout-core/src/ErrorHandler.ts

```diff
@@ -78,22 +132,27 @@ export class ErrorHandler implements ErrorHandlerModel, ObjectWithType {
       if (this._isIgnorableScriptError(errorMessage, fileName, lineNumber, columnNumber, error)) {
         this.handleErrorInfo({
           log: `Ignoring error. Message: ${errorMessage}`,
+          fromGlobalListener: true,
           level: LogLevel.INFO
         });
         return;
       }
       if (error instanceof Error) {
-        this.handle(error)
-          .catch(error => {
-            console.error('Error in global JavaScript error handler', error);
-          });
+        this.analyzeError(error)
+          .then(info => {
+            info.fromGlobalListener = true;
```

Comment:
I am not sure if this flag is correct. If the catch handler in App.ts catches a promise exception, this will be false. Previously, it showed a message box, now there will be a console error only, see ticket.